common/djangoapps/enrollment/views.py

```diff
@@ -725,15 +727,14 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
         **Query Parameters for GET**
 
             * course_id: Filters the result to course enrollments for the course corresponding to the
-              given course ID. The value must be URL encoded. Optional. Required if the 'username' parameter
-              is omitted.
+              given course ID. The value must be URL encoded. Optional.
 
             * username: List of comma-separated usernames. Filters the result to the course enrollments
-              of the given users. Optional. Required if the 'course_id' parameter is omitted.
+              of the given users. Optional.
 
-            * page_size: Number of results to return per page.
+            * page_size: Number of results to return per page. Optional.
```

Comment:
Could you indicate the default page size if this parameter is not passed?  Max page size too?

Reply:
👍